policykit/policyengine/views.py

```diff
@@ -242,6 +240,11 @@ def disable_integration(request):
     if community.platform == name:
         return redirect("/main/settings?error=not_permitted")
 
+    # Temporary: disallow disabling the plugins that have a corresponding PlatformCommunity.
+    # We would need to delete the SlackCommunity as well, which we should show a warning for!
+    if community.platform == "slack":
+        return redirect("/main/settings?error=not_permitted")
```

Comment:
Ok, this is another edge case I'm putting off for now. 😓 

The edge case is:
1. User installs PolicyKit to Discord from /login page, which creates a new Community and a new DiscordCommunity
2. User adds Slack integration from /settings page, which creates a new SlackCommunity linked to an existing Community
3. User clicks "Delete (Slack) Integration" from /settings page while logged in with Discord.

I didn't want to just do `SlackCommunity.delete()` because the user could unintentionally lose tons of data. For now I just disallowed deleting the integration. fyi @amyxzhang

Reply:
Yeah makes sense in the future to put a major warning here.